Sources/Testing/Support/FileHandle.swift

```diff
@@ -140,7 +140,7 @@ struct FileHandle: ~Copyable, Sendable {
       let fd: CInt = -1
 #endif
 
-      if fd >= 0 {
+      if Bool(fd >= 0) {
```

Comment:
This file doesn't build for WASI, however there's a dead-code warning here on platforms where `fileno()` isn't defined which is redundant in the face of our explicit warning, so this change silences it.